src/types/MobileAdsModule.ts

```diff
@@ -0,0 +1,48 @@
+import { RequestConfiguration } from './RequestConfiguration';
+
+/**
+ * An immutable snapshot of a mediation adapter's initialization status.
+ */
+export type AdapterStatus = {
+  name: string;
+  description: string;
+  status: 0 | 1;
```

Comment:
what do the 0 and 1 mean, semantically? Would be nice to have an enum that conveyed semantic meaning in a human way